app/controllers/api/subcollections/tags.rb

```diff
@@ -10,6 +10,15 @@ def assign_tags_resource(type, id, data)
         action_result(false, err.to_s)
       end
 
+      def unassign_tags_resource(type, id, data)
+        resource = resource_search(id, type, collection_class(type))
+        data['tags'].collect do |tag|
+          tags_unassign_resource(resource, type, tag['id'], tag)
+        end
+      rescue => err
```

Comment:
This can be done in a follow-up PR to handle exceptions for both assign_tags and unassign_tags (which have now parity implementation) and return the appropriate failed action_result for the individual tag and not the whole lot.  Thanks.